sourcemash/api/items.py

```diff
@@ -17,18 +16,46 @@
     'category_1': fields.String,
     'category_2': fields.String,
     'summary': fields.String,
-    'category_1': fields.String,
-    'category_2': fields.String,
+    'totalVotes': fields.Integer,
```

Comment:
Let's rename this to score, or points, or something else that reflects that this is a fluctuating sum, not a count of number of votes.